### ISSUE TITLE
(Add option to) Disable smoothing in Month view

### ISSUE THREAD

**Issue by AndreiMiculita** [gnome-pomodoro/gnome-pomodoro#659]
## Is your proposal related to a problem?
It's very difficult to tell which day corresponds to which number of hours in the month view:
![image](https://github.com/gnome-pomodoro/gnome-pomodoro/assets/6352172/37ce41ed-e47e-4c74-8968-1fd1389e1813)
It is also not possible to get more info by hovering over the graph.

## Describe the solution you'd like
I believe the smoothing of the graph should be disabled. Trends would still be visible without it.
A bar chart, with some slight separation between the bars, would be ideal.
Maybe the bars that correspond to the days in the ticks should be a different color (e.g. in the screenshot I provided, the bar for 22 would look different).

**Comment by kamilprusko** [comment]
True, it's not that readable. Disabling smoothing + adding hovers for 31 bars won't be optimal either in my opinion. I'm more leaning on changing this into a heatmap:

![Z9VvW-1022270530](https://github.com/gnome-pomodoro/gnome-pomodoro/assets/1135423/9f4a0d6d-7311-483d-8045-b75b057939c5)

(no mockup yet)

I'll be redoing this view as part of port to GTK4 over the summer.

**Comment by AndreiMiculita** [comment]
Note that I was referring to just disabling smoothing. 

The hover effect is just something I've seen used to make (smoothed) graphs more readable. e.g. on [Google Trends](https://trends.google.com/trends/explore?date=now%201-d&q=gnome&hl=en)

**Comment by kamilprusko** [comment]
Ditching the timeline and using a heatmap only could look like this:

![g125673](https://github.com/gnome-pomodoro/gnome-pomodoro/assets/1135423/fec61ead-bba2-4180-affc-32632146991b)
